### ISSUE TITLE
no SigmoidScaler

### ISSUE THREAD

**Issue by SavelevaV** [DynamicsAndNeuralSystems/pyhctsa#75]
Hi, 

I was running the tutorial file tutorial_bonn_eeg.ipynb, and found an error, in utils.py there is no SigmoidScaler
Line in the code : from pyhctsa.utils import SigmoidScaler

Thanks, 
Best,
Vik

**Comment by joshuabmoore** [comment]
Hi @SavelevaV,

The Sigmoid rescaling transformer was removed in later versions of pyhctsa. For the purposes of this tutorial, a simple MinMax scaler is sufficient. I'll update the notebooks to drop this and use the MinMax scaler instead.

You can simply **comment out** the following lines:
```Python
ss = SigmoidScaler(do_scale=True)
ss_fit = ss.fit(good_features_df)
good_features_df_N = ss.transform(good_features_df)
```

And then __uncomment__ this line:
```Python
good_features_df_N = MinMaxScaler().fit_transform(good_features_df)
```

In the classification problem, you will also need to swap out the SigmpoidScaler for the MinMaxScaler in the loop:
```Python
ss = SigmoidScaler(do_scale=False).fit(X_train)
```
for 
```Python
ss = MinMaxScaler().fit(X_train)
```

**Comment by joshuabmoore** [comment]
I've since updated the Bonn EEG notebook, so this should be fixed now :)
